libs/sdk-ui-dashboard/src/model/events/alerts.ts

```diff
@@ -0,0 +1,202 @@
+// (C) 2021 GoodData Corporation
+import { IWidgetAlert, IWidgetAlertDefinition } from "@gooddata/sdk-backend-spi";
+import { IDashboardEvent } from "./base";
+import { DashboardContext } from "../types/commonTypes";
+
+/**
+ * This event is emitted after the Kpi alert is successfully saved.
+ *
+ * @internal
+ */
+export interface DashboardAlertCreateSuccess extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.CREATE.SUCCESS";
+    readonly payload: {
+        readonly alert: IWidgetAlert;
+    };
+}
+
+export function alertCreateSuccess(
+    ctx: DashboardContext,
+    alert: IWidgetAlert,
+    correlationId?: string,
+): DashboardAlertCreateSuccess {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.CREATE.SUCCESS",
+        ctx,
+        correlationId,
+        payload: {
+            alert,
+        },
+    };
+}
+
+//
+//
+//
+
+/**
+ * This event is emitted after the Kpi alert saving failed.
+ *
+ * @internal
+ */
+export interface DashboardAlertCreateError extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.CREATE.ERROR";
+    readonly payload: {
+        readonly alert: IWidgetAlertDefinition;
+        readonly error: any;
+    };
+}
+
+export function alertCreateError(
+    ctx: DashboardContext,
+    alert: IWidgetAlertDefinition,
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    error: any,
+    correlationId?: string,
+): DashboardAlertCreateError {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.CREATE.ERROR",
+        ctx,
+        correlationId,
+        payload: {
+            alert,
+            error,
+        },
+    };
+}
```

Comment:
this one is not needed; the generic event already can contain error etc. if we need to include detail about the command that failed in that generic event, we can enhance the event and include the entire cmd in it.